types/readable-stream/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: https://github.com/nodejs/readable-stream
 // Definitions by: TeamworkGuy2 <https://github.com/TeamworkGuy2>
 //                   markdreyer <https://github.com/markdreyer>
+//                   benallfree <https://github.com/benallfree>
```

Comment:
Are you intending to become an owner of these packages? You don't need to put your username here if you are juts looking for contribution credit.

Reply:
Nope just wanting to leave a breadcrumb trail in case anyone wonders about the change. I removed this